File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/java/practicas3/biografia.java
package ADSO2928707.portafolio_aprendices.keiner_lindarte_aro.instructor_henry.mis_programas.practicas.java.practicas3;

import java.util.Scanner;

public class biografia {

    public static void main(String[] args) {
        
        try (Scanner input = new Scanner (System.in)){

        System.out.println("Cuantas personas quieren dar a conocer su vida:");
        int cuantos = input.nextInt();

        var relleno = input.nextLine();
      
        for (int vueltas = 1; vueltas <= cuantos; vueltas++) {
      
            System.out.println("----------------------------------------------------------");
      
            System.out.println("!!ANTES QUE TODO DIGITAR LA INFORMACION REQUERIDA!!");
      
            System.out.println("----------------------------------------------------------");
      
            System.out.println("Nombres y apellidos:");
            String name = input.nextLine();
        
            System.out.println("Edad:");
            String age = input.nextLine();
        
            System.out.println("Ahora si porfavor diganos como fue su vida:");
            String life = input.nextLine();
        
            System.out.println("----------------------------------------------------------");    
        
            System.out.println("INFORMACION PROCESADA Y ORDENADA DEL N: $vueltas");
        
            System.out.println("----------------------------------------------------------");
        
            System.out.println("La persona identifica como "+name+" de "+age+" años, nos va a relatar hoy su vida acontinuacion."+relleno);
            System.out.println(" ");
            System.out.println("BIOGRAFIA:");
            System.out.println(" ");
            System.out.println(life);
        
            System.out.println("----------------------------------------------------------");
        
        }
        
        }


    }
    
}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/python/practicas3/biografia.py

print( "Cuantas personas quieren dar a conocer su vida:")
cuantos = int(input())

for vueltas in range(1,cuantos + 1):

    print( "----------------------------------------------------------")

    print( "!!ANTES QUE TODO DIGITAR LA INFORMACION REQUERIDA!!")

    print( "----------------------------------------------------------")

    print( "Nombres y apellidos:")
    name = input()

    print( "Edad:")
    age = int(input())

    print( "Ahora si porfavor diganos como fue su vida:")
    life = input()

    print( "----------------------------------------------------------")

    print( "INFORMACION PROCESADA Y ORDENADA DEL N: ",vueltas)

    print( "----------------------------------------------------------")

    print( "La persona identifica como ",name," de ",age," años, nos va a relatar hoy su vida acontinuacion.")
    print( " ")
    print( "BIOGRAFIA:")
    print( " ")
    print( life)

    print( "----------------------------------------------------------")